app/_locales/en/messages.json

```diff
@@ -2515,9 +2515,8 @@
   "transactionCreated": {
     "message": "Transaction created with a value of $1 at $2."
   },
-  "transactionDetailDappGasHeading": {
-    "message": "$1 suggested gas fee",
-    "description": "$1 represents a dapp origin"
+  "transactionDetailDappGasMoreInfo": {
+    "message": "Site suggested"
```

Comment:
Wondering if we want this to be "dapp suggested" or something along those lines?

Reply:
@rachelcope ^